src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -0,0 +1,189 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React, {useEffect, useMemo, useState} from 'react';
+import {FlatList, View} from 'react-native';
+import * as Illustrations from '@components/Icon/Illustrations';
+import MenuItem from '@components/MenuItem';
+import Section from '@components/Section';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import * as OptionsListUtils from '@libs/OptionsListUtils';
+import * as ReportUtils from '@libs/ReportUtils';
+import type {CentralPaneNavigatorParamList} from '@navigation/types';
+import withPolicy from '@pages/workspace/withPolicy';
+import type {WithPolicyProps} from '@pages/workspace/withPolicy';
+import WorkspacePageWithSections from '@pages/workspace/WorkspacePageWithSections';
+import type {Styles} from '@styles/index';
+// eslint-disable-next-line no-restricted-imports
+import spacing from '@styles/utils/spacing';
+import * as Policy from '@userActions/Policy';
+import CONST from '@src/CONST';
+import type SCREENS from '@src/SCREENS';
+import ToggleSettingOptionRow from './ToggleSettingsOptionRow';
+import type {ToggleSettingOptionRowProps} from './ToggleSettingsOptionRow';
+
+type WorkspaceWorkflowsPageProps = WithPolicyProps & StackScreenProps<CentralPaneNavigatorParamList, typeof SCREENS.WORKSPACE.WORKFLOWS>;
+
+function WorkspaceWorkflowsPage({policy, route}: WorkspaceWorkflowsPageProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const {isSmallScreenWidth} = useWindowDimensions();
+    const theme = useTheme();
+
+    const [policyOwnerDisplayName, setPolicyOwnerDisplayName] = useState('');
+
+    useEffect(() => {
+        if (!policy?.ownerAccountID) {
+            return;
+        }
+        const ownerPersonalDetails = ReportUtils.getDisplayNamesWithTooltips(OptionsListUtils.getPersonalDetailsForAccountIDs([policy.ownerAccountID], CONST.EMPTY_OBJECT), false);
+        if (ownerPersonalDetails.length > 0 && ownerPersonalDetails[0].displayName) {
+            setPolicyOwnerDisplayName(ownerPersonalDetails[0].displayName);
+        }
+    }, [policy]);
+
+    // Since these styles are only used in this component we define them here
```

Comment:
Same here

Reply:
Removed it and reused app styles, see commit [here](https://github.com/Expensify/App/pull/36345/commits/8b5e067d1f41535ebb093e0832d39ee22973bd81)